### DISCUSSION TITLE
Import de données ?

### DISCUSSION THREAD

**Discussion by frgith** [dbarzin/mercator#643] [Q&A]
Bonjour, 
Est ce qu'il est possible d'importer des données dans mercator ?
Par exemple les informations des vues logique ou infrastructure si on les a sous forme de fichier (csv, excel ...)
Cordialement, 
Fabrice

**Comment by dbarzin** [comment]
Bonjour,

Il existe une [API](https://dbarzin.github.io/mercator/api/) avec des exemples d'utilisation dans différents langages.

Il est possible également d'importer des données avec la commande [load data](https://dev.mysql.com/doc/refman/8.0/en/load-data.html) ou avec [MySQL Workbench](https://dev.mysql.com/doc/workbench/en/wb-admin-export-import-table.html) ou encore [Heidi SQL](https://www.heidisql.com/help.php#import).

Il n'est pas prévu de le faire via l'application.

Didier

**Comment by dbarzin** [comment]
Vous pouvez désormais importer des données à partir d’une feuille de calcul Excel directement depuis le menu Configuration → Import !
Un moyen simple et rapide pour initialiser ou mettre à jour vos référentiels ✨
